application/models/QuestionBaseRenderer.php

```diff
@@ -59,8 +59,9 @@ public function __construct($aFieldArray, $bRenderDirect = false)
         $oQuestionTemplate = QuestionTemplate::getNewInstance($this->oQuestion);
         $oQuestionTemplate->registerAssets(); // Register the custom assets of the question template, if needed
 
-        if (!empty($this->oQuestion->questionl10ns[$this->sLanguage]->script)) {
-            $sScriptRendered = LimeExpressionManager::ProcessString($this->oQuestion->questionl10ns[$this->sLanguage]->script, $this->oQuestion->qid, ['QID' => $this->oQuestion->qid]);
+        $questionScript = $this->getQuestionScript();
+        if (!empty($questionScript)) {
+            $sScriptRendered = LimeExpressionManager::ProcessString($questionScript, $this->oQuestion->qid, ['QID' => $this->oQuestion->qid]);
```

Comment:
We get dynamic script here ? Must use only static expression, no ? (another issue)

Reply:
@tpartner see issue https://bugs.limesurvey.org/view.php?id=18024 about Script in static or dynamic :)